src/plugins/analysis/users_and_passwords/view/users_and_passwords.html

```diff
@@ -12,7 +12,7 @@
                     <table class="table table-bordered mb-0">
                     {% for item in firmware.processed_analysis[selected_analysis][key] | sort %}
                         {% if item not in ["log", "cracked"] %}
-                            <tr{% if item == 'password' %} class="table-danger"{% endif %}>
+                            <tr{% if item == 'password'%} class="table-danger"{% endif %}{% if item == 'ERROR'%} class="table-warning"{% endif %}>
```

Comment:
the `{% endif %}{% if item == 'ERROR'%}` could probably be shortened to `{% elif item == 'ERROR'%}`